tests/kernel/interrupt/src/interrupt_offload.c

```diff
@@ -0,0 +1,266 @@
+/*
+ * Copyright (c) 2021 Intel Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <ztest.h>
+#include <irq_offload.h>
+#include <interrupt_util.h>
+
+#define STACK_SIZE	1024
+#define NUM_WORK	4
+
+static struct k_work offload_work[NUM_WORK];
+static struct k_work_q wq_queue;
+static K_THREAD_STACK_DEFINE(wq_stack, STACK_SIZE);
+static K_THREAD_STACK_DEFINE(tstack, STACK_SIZE);
+static struct k_thread tdata;
+
+static struct k_sem start_sem;
+static struct k_sem end_sem;
+
+/*
+ * This global variable control if the priority of offload job
+ * is greater than the original thread.
+ */
+static bool offload_job_prio_higher;
+
+static volatile int orig_t_keep_run;
+
+enum {
+	TEST_OFFLOAD_MULTI_JOBS,
+	TEST_OFFLOAD_INDETICAL_JOBS
```

Comment:
Here is a misprint, IDENTICAL instead INDETICAL.

Reply:
Thanks for correcting this. Got it fixed.